extensions/chocolatey-core.extension/extensions/Install-ChocolateyInstallPackage.ps1

```diff
@@ -0,0 +1,104 @@
+.PARAMETER UseOnlyPackageSilentArguments
+Do not allow choco to privde/merge additional silent arguments and
+only use the ones available with the package. Available in 0.9.10+.
+.PARAMETER IgnoredArguments
+Allows splatting with argumnets that do not apply. Do not use directly.
+.EXAMPLE
+$packageName= 'bob'
+$toolsDir   = "$(Split-Path -Parent $MyInvocation.MyCommand.Definition)"
+$fileLocation = Join-Path $toolsDir '32BIT_INSTALLER_EMBEDDED_IN_PACKAGE'
+$fileLocation64 = Join-Path $toolsDir '64BIT_INSTALLER_EMBEDDED_IN_PACKAGE'
+
+$packageArgs = @{
+  packageName   = $packageName
+  fileType      = 'msi'
+  file          = $fileLocation
+  file64        = $fileLocation64
+  silentArgs    = "/qn /norestart"
+  validExitCodes= @(0, 3010, 1641)
+  softwareName  =
+}
+
+Install-ChocolateyInstallPackageEx @packageArgs
+#>
+$have64BitFileParam = Get-Item function:\Install-ChocolateyInstallPackage | ? { $_.Parameters.ContainsKey('file64') }
+if ($have64BitFileParam) {
+  return
+}
+
+Write-Debug "Loading Install-ChocolateyInstallPackage override"
+Rename-Item function:Install-ChocolateyInstallPackage Install-ChocolateyInstallPackageOriginal
+
+function Install-ChocolateyInstallPackage {
+  param(
+    [parameter(Mandatory=$true, Position=0)][string] $packageName,
+    [parameter(Mandatory=$false, Position=1)]
+    [alias("installerType","installType")][string] $fileType = 'exe',
+    [parameter(Mandatory=$false, Position=2)][string[]] $silentArgs = '',
+    [alias("fileFullPath")][parameter(Mandatory=$false, Position=3)][string] $file,
+    [alias("fileFullPath64")][Parameter(Mandatory=$false, Position=4)][string] $file64,
+    [parameter(Mandatory=$false)] $validExitCodes = @(0),
+    [parameter(Mandatory=$false)]
+    [alias("useOnlyPackageSilentArgs")][switch] $useOnlyPackageSilentArguments = $false,
+    [parameter(ValueFromRemainingArguments = $true)][Object[]] $ignoredArguments
+  )
+  if (!$file -and !$file64) {
+    throw "No file path have been specified."
+  }
+
+  $fileFullPath = $null
+
+  if ((Get-ProcessorBits 32) -or $env:chocolateyForceX86 -eq "$true") {
+    if (!$file) {
+      throw "32-bit installation is not supported for $packageName"
+    } elseif($file64) {
+      Write-Host "Installing $packageName 32-bit"
+    }
+    $fileFullPath = $file
+  }else {
+    if (!$file64) {
+      $fileFullPath = $file
+    } else {
+      Write-Host "Installing $packageName 64-bit"
+    }
+    $fileFullPath = $file64
+  }
+
+  $packageArgs = @{
+    packageName    = $packageName
+    fileType       = $fileType
+    silentArgs     = $silentArgs
+    file           = $fileFullPath
+    validExitCodes = $validExitCodes
+    useOnlyPackageSilentArgs = $useOnlyPackageSilentArguments
+    ignoredArguments = $ignoredArguments
+  }
+
+  Install-ChocolateyInstallPackageOriginal @packageArgs
+}
+
+# We need to force the export of Install-ChocolateyInstall function
+Export-ModuleMember -Function Install-ChocolateyInstallPackage
```

Comment:
I believe so yes (I did try it on my own computer, although only through importing `chocolatey-core.psm1` manually).
Since the `chocolatey-core.psm1` is sourcing all of the files, it just don't export it since it had found an existing function with the same name.